modules/twinkleblock.js

```diff
@@ -430,27 +430,67 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 
 		$form.find('[name=pagerestrictions]').select2({
 			width: '100%',
-			tags: true,
-			placeholder: 'Enter pages to block user from',
+			placeholder: 'Select pages to block user from',
```

Comment:
```suggestion
			placeholder: 'Select pages to block user from',
			language: {
				errorLoading: function() { return 'Incomplete or invalid search term'; }
			},
```

Or something like that.  Any page outside the article namespace will trigger this message; the delay should help, but if there's any pause after the colon, the user will see "The results could not be loaded", which (to me anyway) suggests an error on Twinkle's part.  It'd be good to cover the (slim) chance that an invalid search term gets entered (` `, `(`, `[`, etc.) as well.